runtime/trappist/src/lib.rs

```diff
@@ -120,7 +123,7 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
-	Migrations,
+	(migrations::SetStorageVersions, pallet_xcm::migration::v1::MigrateToV1<Runtime>),
```

Comment:
General question, any harm in retaining the `pallet_contracts::Migration<Runtime>` from the previous config?

Reply:
i think that best practice is to leave the `Migrations` type empty if there are not migrations and check for pallet-contracts if they contain any migration in next releases. 

Do you suggest leaving them for every upgrade so contracts migrations are executed automatically in case there are new migrations?

Reply:
> Perhaps the PRA should call out pallet migrations in a separate section explicitly? I did a quick check on the last few and saw that while it talks about general improvements to migrations, if doesnt seem to specify pallet-specific migrations. Could that be useful?

i was thinking exactly to propose the same. PRAs should have a section only for FRAME and system pallets storage migrations.

CC: @al3mart